GUI/QtQuick/Utils.cpp

```diff
@@ -0,0 +1,20 @@
+#include "Utils.h"
+#include "BTCNumericTypes.h"
+
+using namespace gui_utils;
+
+QString gui_utils::balance_to_qstring(float balance)
+{
+   if (balance < 0) {
+      balance = 0;
+   }
+   return normalized_balance_to_qstring(balance / BTCNumericTypes::BalanceDivider);
+}
+
+QString gui_utils::normalized_balance_to_qstring(float balance)
```

Comment:
balance should be `double` here
also remember the camelCase naming convention in C++ code